LA4Ice.c

```diff
@@ -283,7 +283,7 @@ int main(int argc, char *argv[])
   }
 
   //  Read the file and display selected records
-  
+  if (tspace > 0)
```

Comment:
I'm afraid that I don't know this code as well as you do. What is `tspace` for? It seems to be only some sort of storage indication. `novl` seems to represent the actual number of overlaps. So why not compare `novl==0` instead, or at least in addition?

But I'm curious how your code makes any difference. If `novl` is `0`, then the for-loop should be a no-op, so this should be fast. If `novl` is *not* `0`, then can `tspace` still be non-zero? I'm thinking yes, but I'm not sure.

If you can definitely answer those questions, then merge. If you're not sure, then I'll look into this more deeply later.